react-app/src/components/OverviewScreen/OverviewScreen.graphql

```diff
@@ -0,0 +1,10 @@
+query OverviewScreenQueryCommunityStatus {
```

Comment:
we can name this `OverviewScreenQuery`? if there is anything extra needed we will combine in one query anyways?

Reply:
can rename later